### PR TITLE
Graph server deploy

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: Jenkinsfile.deploy [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@Library(value='PipelineLibs@0.6.1', changelog=false) _
```
Comment (ssoper-usgs) [inline Jenkinsfile.deploy]: Might want to use `0.6.3` here to mirror the library version in the build file.

File: graph-server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Add manifest and install dependencies
+COPY package.json package-lock.json ./
+RUN npm ci
+
+# Add the server source code
+COPY ./src ./src
+
+# Add customer entrypoing that will help us set environment variables in ECS
+COPY ./bin/docker-entrypoint.sh /usr/local/bin/
+ENTRYPOINT ["docker-entrypoint.sh"]
```
Comment (ssoper-usgs) [inline graph-server/Dockerfile]: Might be different for node builds, but do we still want to use `entrypoint` here?

File: Jenkinsfile.deploy (hunk 1 above)
Comment (ayan-usgs) [reply Jenkinsfile.deploy]: > Might want to use `0.6.3` here to mirror the library version in the build file.

I'm actually thinking it's going to be the next version (e.g. `0.7.0`) that we'll be using, so I left as Dan had it. The next library version has a new parameter that's important for the server to work.

File: graph-server/Dockerfile (hunk 2 above)
Comment (ayan-usgs) [reply graph-server/Dockerfile]: As of right now, I think so.

**Pull request merged**